manager/integration/tests/test_node.py

```diff
@@ -0,0 +1,375 @@
+    update_disk.append(disk2)
+
+    # save disks to node
+    node = node.diskUpdate(disks=update_disk)
+    assert len(node["disks"]) == len(update_disk)
+
+    node = client.by_id_node(lht_hostId)
+    assert len(node["disks"]) == len(update_disk)
+
+    # update disk
+    disks = node["disks"]
+    update_disk = get_update_disks(disks)
+    for disk in update_disk:
+        # keep default disk for other tests
+        if disk["path"] == disk_path1 or disk["path"] == disk_path2:
+            disk["allowScheduling"] = False
+            disk["storageReserved"] = Gi
+    node.diskUpdate(disks=update_disk)
+    # wait for node controller to update disk status
+    time.sleep(WAIT_FOR_NODE_STATUS)
+    node = client.by_id_node(lht_hostId)
+    disks = node["disks"]
+    for key, disk in disks.iteritems():
+        if disk["path"] == disk_path1:
+            assert not disk["allowScheduling"]
+            assert disk["storageReserved"] == Gi
+            assert disk["storageScheduled"] == 0
+            assert disk["storageMaximum"] == 5*Gi
+            free, total = common.get_host_disk_size(disk_path1)
+            assert disk["storageAvailable"] == free
+        elif disk["path"] == disk_path2:
+            assert not disk["allowScheduling"]
+            assert disk["storageReserved"] == Gi
+            assert disk["storageScheduled"] == 0
+            free, total = common.get_host_disk_size(disk_path2)
+            assert disk["storageMaximum"] == total
+            assert disk["storageAvailable"] == free
+
+    # delete other disks, just remain default disk
+    update_disk = get_update_disks(disks)
+    remain_disk = []
+    for disk in update_disk:
+        # keep default disk for other tests
+        if disk["path"] != disk_path1 and disk["path"] != disk_path2:
+            remain_disk.append(disk)
+    node = node.diskUpdate(disks=remain_disk)
+    assert len(node["disks"]) == len(remain_disk)
+    # clear disks
+    clean_disk(client, 'vol-disk-1', 'vol-disk-2')
+
+
+def create_volume(client, vol_name, size, node_id, r_num):
+    volume = client.create_volume(name=vol_name, size=size,
+                                  numberOfReplicas=r_num)
+    assert volume["numberOfReplicas"] == r_num
+    assert volume["frontend"] == "blockdev"
+
+    volume = common.wait_for_volume_detached(client, vol_name)
+    assert len(volume["replicas"]) == r_num
+
+    assert volume["state"] == "detached"
+    assert volume["created"] != ""
+
+    volumeByName = client.by_id_volume(vol_name)
+    assert volumeByName["name"] == volume["name"]
+    assert volumeByName["size"] == volume["size"]
+    assert volumeByName["numberOfReplicas"] == volume["numberOfReplicas"]
+    assert volumeByName["state"] == volume["state"]
+    assert volumeByName["created"] == volume["created"]
+
+    volume.attach(hostId=node_id)
+    volume = common.wait_for_volume_healthy(client, vol_name)
+
+    return volume
+
+
+def clean_volume(client, vol_name):
```

Comment:
`cleanup_volume`.

Also, you don't need to wait for the volume to detach before delete it.